chia/rpc/wallet_rpc_api.py

```diff
@@ -541,7 +572,27 @@ async def create_new_wallet(self, request: Dict):
                 }
             else:  # undefined did_type
                 pass
-
+        elif request["wallet_type"] == "nft_wallet":
+            for wallet in self.service.wallet_state_manager.wallets.values():
+                if wallet.type() == WalletType.NFT:
```

Comment:
We can only have one NFT wallet for each key? What is the rationale for this?

Reply:
We can only have one "standard nft wallet" per pubkey and one per DID (NFT1 version we're working on). One pubkey can have multiple DIDs. 
Standard NFT wallet will receive any incoming NFTs that go to standard wallet puzzlehash and don't have a DID assigned, so acts as a default.